definitions/npm/ansi-regex_v4.x.x/test_ansi-regex.js

```diff
@@ -0,0 +1,29 @@
+import ansiRegex from 'ansi-regex';
+import { describe, it } from 'flow-typed-test';
+
+describe('ansi-regex', () => {
+  it('should be RegExp', () => {
+    ansiRegex(); // $ExpectType RegExp
+  });
+
+  it('should be boolean', () => {
+    // From the ansi-regex README.md
+    ansiRegex().test('\u001B[4mcake\u001B[0m'); // $ExpectType boolean
+    // => true
+  });
+
+  it('should be false boolean', () => {
+    ansiRegex().test('cake'); // $ExpectType boolean
```

Comment:
You can test types like `(ansiRegex().test('cake'): boolean)` or `const bool: boolean = ansiRegex().test('cake');`